caravel/helpers.py

```diff
@@ -65,7 +71,7 @@ def glob_if_exists(x):
     :param x: a string with path containing globs
     :return list[str]: a list of paths
     """
-    return [glob.glob(e) or e for e in x] if coll_like(x) else (glob.glob(x) or [x])
+    return [glob.glob(e) or e for e in x] if peppy.utils.coll_like(x) else (glob.glob(x) or [x])
```

Comment:
Nice! It's cool that we're really starting to separate and reuse things.

Reply:
Yes. Some util-like functions are repeated in some of our packages and I had a hard time figuring out which version is the the most recent one. 

It would be nice to have them centralized in a standalone package/repo

Reply:
are they attmap-like enough to be moved to attmap?

Reply:
> Yes. Some util-like functions are repeated in some of our packages and I had a hard time figuring out which version is the the most recent one.
> 
> It would be nice to have them centralized in a standalone package/repo

Yeah, there's not been a single effort to move everything, so things will probably remain a bit fragmented and sometimes repeated, though diminishingly so =D

Reply:
> are they attmap-like enough to be moved to attmap?

If we move some stuff, I'd prefer to make a just-utils sort of package. `attmap` I guess is more general than `peppy`, but we have plenty of helpers that aren't tied to `attmap` either, just collections or other kinds of types more broadly.